### DISCUSSION TITLE
开启tailwindcss，编辑器不提示类名

### DISCUSSION THREAD

**Discussion by hjldare** [umijs/umi#9111] [General]
编辑器：webstorm
问题：cli开启tailwincss后，写className不提示

**Comment by fz6m** [comment]
需要安装 tailwind 的 webstorm 插件。自己搜索下。

**Reply by fz6m** [reply]
https://www.jetbrains.com/help/webstorm/tailwind-css.html

**Reply by hjldare** [reply]
谢谢，不过webstorm是默认安装tailwind插件的。上面没说清楚是umi max项目开启tailwind不提示,而是提示antd的css变量。如果不创建antd pro项目开启tailwind是提示正确的
